pom.xml

```diff
@@ -459,6 +459,7 @@
     <version.javax.inject>1</version.javax.inject>
     <version.javax.servlet>3.0.1</version.javax.servlet>
     <version.commons-httpclient>3.1-jbossorg-1</version.commons-httpclient>
+    <version.commons-io>2.8.0</version.commons-io>
```

Comment:
This is required for the seamless inclusion in Quarkus and TT. The resteasy client doesn't bring this dependency and it fails some transformations in MessageBodyReaders and Writers. I was hesitant to include this also but take it as a short term workaround that we should try to fix in RESTEasy.

Reply:
I don't follow, if a dependency is not used it does not need to be included.

Reply:
@xstefank you have not addressed this comment yet. Also if no other module needs a property then it would need to go in the appropriate pom (this one is the top level narayana pom).

Reply:
ok, I'll move the version. This is is both with `resteasy-json-binding-provider` required to make lra-client work with resteasy in Quarkus and TT because they do package jackson MessageBodyReaders and MessageBodyWriters in a separate extensions or in the case of TT in a separate fractions. I thought that this is unnecessary if we can provide the reader/writer manually.

Reply:
Wait. Is this a runtime dependency? I will try running the test suite on your PR ...

Reply:
yes, that is runtime. But it won't be affecting the test suite. Only if you add lra-client dependency to the newly generated Quarkus or TT application.